openpype/hosts/blender/plugins/publish/collect_review.py

```diff
@@ -39,7 +39,7 @@ def process(self, instance):
 
         if not instance.data.get("remove"):
 
-            task = legacy_io.Session.get("AVALON_TASK")
+            task = get_current_task_name()
```

Comment:
Like metioned in description, in some cases I didn't use `context.data` because I would have to change order of the plugin.

Reply:
But context.data task collection always runs at start right? It's at CollectorOrder - 0.5?

Reply:
> It's at CollectorOrder - 0.5?

Yes. And because any order before that means that the plugin is ignored I would have to change order of this plugin.

Reply:
Sorry, I don't understand. This plug-in runs at `CollectorOrder + 0.3` and global plugin CollectCurrentContext which sets `context.data["task"]` runs at `CollectorOrder - 0.5`. As such , this should be able to be `task = context.data["task"]` right?

Reply:
Nothing I thought this is different plugin.